src/pages/Address/Address.js

```diff
@@ -76,11 +90,24 @@ class Address extends React.Component {
             )}
           </div>
           <div className="searchResult">
-            <p>
-              고객님은 <span className="text-bold">당일배송</span>,
-              <span className="text-bold">새벽배송</span>으로
-              <br /> 받아보실 수 있습니다.
-            </p>
+            {isDelivery ? (
+              <p>
+                고객님은 <span className="text-bold">당일배송</span>,
+                <span className="text-bold">새벽배송</span>으로
+                <br /> 받아보실 수 있습니다.
+              </p>
+            ) : (
+              ''
+            )}
+            {isDelivery === false ? (
+              <p>
+                <span className="text-bold">신선배송 불가지역이므로 </span>,
+                <span className="red">우체국 택배</span>로
+                <br /> 받아보실 수 있습니다.
+              </p>
+            ) : (
+              ''
+            )}
```

Comment:
- 하나의 삼항 연산자로 처리할 수 있습니다!
```suggestion
            {isDelivery ? (
              <p>
                고객님은 <span className="text-bold">당일배송</span>,
                <span className="text-bold">새벽배송</span>으로
                <br /> 받아보실 수 있습니다.
              </p>
            ) : (
              <p>
                <span className="text-bold">신선배송 불가지역이므로 </span>,
                <span className="red">우체국 택배</span>로
                <br /> 받아보실 수 있습니다.
              </p>
            )}
```

Reply:
수정 완료했습니다!